test/parallel/test-fs-whatwg-url.js

```diff
@@ -63,7 +63,7 @@ if (common.isWindows) {
       code: 'ERR_INVALID_ARG_VALUE',
       type: TypeError,
       message: 'The argument \'path\' must be a string or Uint8Array without ' +
-               'null bytes. Received \'c:/tmp/\\u0000test\''
+               'null bytes. Received \'c:\\\\tmp\\\\\\u0000test\''
```

Comment:
Updated this test after I noticed the error.

The reason for multiple backslashes is that ERR_INVALID_ARG_VALUE calls `inspect` on the invalid argument value.